docs/specs/clients/sign/rpc-methods.md

```diff
@@ -338,3 +342,87 @@ true
 | TTL     | 30       |
 | Tag     | 1115     |
 ```
+
+## Wallet Authentication RPC Methods
+
+### Definitions
+
+- **Nullables:** Fields flagged as `Optional` can be omitted from the payload.
+- Unless explicitly mentioned that a response requires associated data, all methods response's follow a default JSON-RPC pattern for the success and failure cases:
+
+```jsonc
+// Success
+result: true
+
+// Failure
+error: {
+  "code": number,
+  "message": string
+}
+```
+
+### Methods
+
+#### wc_sessionAuthenticated
+
+Used to request authentication signature to a peer through topic P. Response is expected on topic R with shared symKey.
+
+Parameters for request and response are compliant with [CAIP-74](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-74.md) (aka CACAO)
+
+- Success response is equivalent to authentication acceptance.
+- Error response is equivalent to authentication rejection.
+
+##### Expiry
+
+Param `Expiry` is an optional Unix timestamp. Sets the time until which the responder can respond to this request. If request is expired responder should respond with a specific error code.
+
+If this parameter is not specified, the request is considered indefinite.
```

Comment:
I suggest setting a default value for Expiry, let's say 5mins instead of considering it indefinite when not specified. What's the value of having a request being indefinite?

Reply:
good idea although I would say to increase the expiry as singinig with MPC wallet would take much longer and proposer can't know in advance. We can have it by default 1d or 7d? Wdyt?

Reply:
1d sounds good

Reply:
1h -> 3600s